### DISCUSSION TITLE
Пример для RecentTracks в документации

### DISCUSSION THREAD

**Discussion by Bedmit** [Chimildic/goofy#10] [Идея] [answered]
Не разобрался куда это лучше, потому напишу в Беседке.
Мне кажется в https://chimildic.github.io/goofy/#/func?id=recenttracks не хватает примера, как в описаниях остальных функций. Без них, несведущему, разобраться вообще нереально.
И заодно.  Если не выставлен лимит, сколько треков берется в этот массив?

**Comment by Chimildic** [comment, marked as answer]
Добавил пример и дополнил описание про лимит на сайте. 
Если не указано, возвращаются все доступные.
